### ISSUE TITLE
Error occurs If jobs are cancelled

### ISSUE THREAD

**Issue by tadashi-aikawa** [homoluctus/slatify#7]
Hi. Thank you for such a beautiful actions!

I found that a error occurs If jobs are cancelled by failure on matrix build.
Console output is below.

```
Run homoluctus/slatify@master
  with:
    type: Cancelled
    job_name: :python:*3.6* e2e tests
    icon_emoji: tio2
    url: ***
    username: Github Actions
  env:
    pythonLocation: /opt/hostedtoolcache/Python/3.6.9/x64
Error: Allow words that contain "success" or "fail"
    at Object.getStatus (/home/runner/work/_actions/homoluctus/slatify/master/lib/utils.js:18:15)
    at /home/runner/work/_actions/homoluctus/slatify/master/lib/main.js:37:36
    at Generator.next (<anonymous>)
    at /home/runner/work/_actions/homoluctus/slatify/master/lib/main.js:7:71
    at new Promise (<anonymous>)
    at __awaiter (/home/runner/work/_actions/homoluctus/slatify/master/lib/main.js:3:12)
    at run (/home/runner/work/_actions/homoluctus/slatify/master/lib/main.js:22:12)
    at Object.<anonymous> (/home/runner/work/_actions/homoluctus/slatify/master/lib/main.js:48:1)
    at Module._compile (internal/modules/cjs/loader.js:774:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:785:10)
##[error]Allow words that contain "success" or "fail"
##[error]Node run failed with exit code 1
```

I think why this occurs is `Cancelled` is not defined in `utils.ts` . (and `neutral`) 

Best regards.

🔗 References
* [Console output](https://github.com/tadashi-aikawa/owcli/commit/394f1190183272f5d3215fdb9ffb5a7c25181bf3/checks)
* [GitHub Actionについて](https://help.github.com/ja/articles/about-github-actions#notifications-for-workflow-runs)
> The notification will include the workflow run's status (including successful, failed, neutral, and canceled runs).

**Comment by homoluctus** [comment]
Thanks @tadashi-aikawa 

We have not developed the notification feature for cancel.
Currently, this app notifies you when job fails or succeeds.

We will add the feature later 💪

**Comment by homoluctus** [comment]
This issue was resolved #9 
Thanks @tadashi-aikawa

**Comment by tadashi-aikawa** [comment]
Thanks for your quick working! 😄
